tests/unit_tests/data_steward/retraction/retract_deactivated_pids_test.py

```diff
@@ -0,0 +1,226 @@
+        date_fields_info_df = pd.DataFrame()
+        dataset_obj = self.mock_bq_client.list_datasets(self.project_id)
+        datasets = [d.dataset_id for d in dataset_obj]
+
+        for dataset in datasets:
+            pids_tables_df = retract_deactivated_pids.get_pids_table_info(
+                self.project_id, dataset, self.mock_bq_client)
+
+            # Keep only records with datatype of 'DATE'
+            date_fields_df = pids_tables_df[pids_tables_df['data_type'] ==
+                                            'DATE']
+
+            # Create df to append to, keeping only one record per table
+            df_to_append = pd.DataFrame(columns=[
+                'project_id', 'dataset_id', 'table', 'date_column',
+                'start_date_column', 'end_date_column'
+            ])
+            df_to_append['project_id'] = date_fields_df['table_catalog']
+            df_to_append['dataset_id'] = date_fields_df['table_schema']
+            df_to_append['table'] = date_fields_df['table_name']
+            df_to_append = df_to_append.drop_duplicates()
+
+            # Create new df to loop through date time fields
+            df_to_iterate = pd.DataFrame(
+                columns=['project_id', 'dataset_id', 'table', 'column'])
+            df_to_iterate['project_id'] = date_fields_df['table_catalog']
+            df_to_iterate['dataset_id'] = date_fields_df['table_schema']
+            df_to_iterate['table'] = date_fields_df['table_name']
+            df_to_iterate['column'] = date_fields_df['column_name']
+
+            # Remove person table and death table
+            df_to_append = df_to_append[~df_to_append.table.str.
+                                        contains('person')]
+            df_to_iterate = df_to_iterate[~df_to_iterate.table.str.
+                                          contains('person')]
+
+            # Filter through date columns and append to the appropriate column
+            for row in df_to_iterate.iterrows():
+                column = getattr(row, 'column')
+                table = getattr(row, 'table')
+                if 'start_date' in column:
+                    df_to_append.loc[df_to_append.table == table,
+                                     'start_date_column'] = column
+                elif 'end_date' in column:
+                    df_to_append.loc[df_to_append.table == table,
+                                     'end_date_column'] = column
+                else:
+                    df_to_append.loc[df_to_append.table == table,
+                                     'date_column'] = column
+
+            date_fields_info_df = date_fields_info_df.append(df_to_append)
+
+        returned_df = retract_deactivated_pids.get_date_info_for_pids_tables(
+            self.project_id, self.mock_bq_client)
+
+        assert_frame_equal(date_fields_info_df, returned_df)
+
+    @mock.patch('utils.bq.query')
+    @mock.patch(
+        'retraction.retract_deactivated_pids.get_date_info_for_pids_tables')
```

Comment:
This unit test is not providing any worthwhile information.  It ends by comparing two blank lists.